torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -1591,6 +1593,106 @@ void TensorExprKernel::bindInput(const torch::jit::Value* input) {
   }
 }
 
+namespace {
+
+std::vector<VarHandle> squeezeIndices(
```

Comment:
A comment here would help a lot in understanding this helper.

Reply:
Done.